cmd/token_test.go

```diff
@@ -89,6 +89,46 @@ func TestTokenWithGCEAK(t *testing.T) {
 	}
 }
 
+func TestCustomEventLogFile(t *testing.T) {
+	if _, err := os.Stat("/dev/tpm0"); os.IsNotExist(err) {
+		t.Skip("Skipping test: /dev/tpm0 not found")
+	}
+
+	ExternalTPM = nil
+	var dummyMetaInstance = util.Instance{ProjectID: "test-project", ProjectNumber: "1922337278274", Zone: "us-central-1a", InstanceID: "12345678", InstanceName: "default"}
```

Comment:
Sorry, one more thing: I see now you mean for this test to run only in GCE environments with TPMs. In that case, another test to add would be to copy the default event log location to a https://pkg.go.dev/testing#T.TempDir and then point the `event-log` flag there. Please post a copy of a successful test run as a comment too.

Reply:
I don't think we run cmd tests in GCE yet, maybe we can manually test it?

Reply:
To copy `eventlog` file, the `go test` would need root privilege. We might not want this?

Reply:
I added a TestCopiedCustomEventLogFile for running manually with the following commands:
`go test -c -v ./cmd/...`
`sudo ./cmd.test -test.run TestCopiedCustomEventLogFile`